lua/wikis/commons/Widget/NavBox/AutoTeam.lua

```diff
@@ -0,0 +1,145 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/NavBox/AutoTeam
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+
+local Class = Lua.import('Module:Class')
+local Array = Lua.import('Module:Array')
+local Logic = Lua.import('Module:Logic')
+local Info = Lua.import('Module:Info', {loadData = true})
+local Table = Lua.import('Module:Table')
+
+local Widget = Lua.import('Module:Widget')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local Link = Lua.import('Module:Widget/Basic/Link')
+
+local TeamService = Lua.import('Module:Service/Team')
+
+local NavBox = Lua.import('Module:Widget/NavBox')
+
+local DEFAULT_LINK_CONFIG = {
+	results = 'Results',
+	playedMatches = 'Played Matches',
+}
+
+---@class AutoTeamNavbox: Widget
+---@operator call(table): AutoTeamNavbox
+local AutoTeamNavbox = Class.new(Widget)
+AutoTeamNavbox.defaultProps = {team = mw.title.getCurrentTitle().prefixedText}
+
+---@return Widget?
+function AutoTeamNavbox:render()
+	local team = TeamService.getTeamByTemplate(self.props.team)
+	if not team then
+		return
+	end
+
+	local members = team.members
+
+	if not members or #members == 0 then
+		return
+	end
+
+	local activeMembers = Array.filter(members, function(member)
+		return member.status == 'active' and Logic.isNotEmpty(member.displayName)
+	end)
+
+	local activePlayers = Array.filter(activeMembers, function(member)
+		return member.type == 'player' and member.isMain
+	end)
+	local secondaryRosterActivePlayers = Logic.nilIfEmpty(Array.filter(activeMembers, function(member)
+		return member.type == 'player' and not member.isMain
+	end))
+	local activeStaff = Array.filter(activeMembers, function(member)
+		return member.type == 'staff'
+	end)
+	local config = Info.config.teamRosterNavbox or {}
+	if config.staffLimitedPositions then
+		activeStaff = Array.filter(activeStaff, function(member)
+			return Table.includes(config.staffLimitedPositions, member.position)
+		end)
+	end
```

Comment:
Let's remove this option

Reply:
if apex laments about it i will blame you :P